CryptoPkg/Library/TlsLibMbedtls/TlsConfig.c

```diff
@@ -479,7 +362,7 @@ TlsSetCompressionMethod (
   IN     UINT8  CompMethod
   )
 {
-  return EFI_UNSUPPORTED;
+  return EFI_SUCCESS;

```

Comment:
Incorrect here, 
if CM is 0, then return success.
if CM is 1, should config a avaliable Compression methon.
Should check whether Mbedtls have similar functions like openssl.